src/VueTyping.js

```diff
@@ -1,58 +1,102 @@
 export default {
-    render (h) {
-        return h('span', this.speechFrameFragment)
-    },
-    props: {
-        framerate: {
-            type: Number,
-            default: 1
+  render(h) {
+    return h("div", [
+      h("span", this.speechFrameFragment),
+      h(
+        "span",
+        {
+          style: {
+            opacity: this.cursorOpacity,
+            color: this.cursorAnimationType.color
+          }
         },
-        text: {
-            type: String,
-            default: ''
-        }
+        this.cursorAnimationType.cursor
+      )
+    ]);
+  },
+  props: {
+    framerate: {
+      type: Number,
+      default: 1
     },
-    data () {
-        return {
-            speechFrameFragment: ''
-        }
+    text: {
+      type: String,
+      default: ""
     },
-    watch: {
-        text (value) {
-            this.scheduleTyping()
-        },
-        framerate () {
-            this.scheduleTyping()
-        }
+    cursorAnimationType: {
+      type: Object,
+      default: {
+        blinking: true,
+        cursor: "|",
+        color: "black",
+        framerate: 24
```

Comment:
Framerate values are hard to understand. Maybe we can add some defaults, like:
```js
// Just arbitrary values, but we can test it in the demo
export const CursorSpeed = {
  slow: 40,
  normal: 24,
  fast: 5
}
```

then the user should be able to do something like this:
```js
import VueTyping, { CursorSpeed }
new Vue({
  template: '<vue-typing :cursorAnimationType="$options.cursorOptions" />',
  cursorOptions: {
    framerate: CursorSpeed.fast
  }
})
```

this is not needed, I just want your opinion for this to be a new feature

Reply:
Yeah, i think that sounds good, i will adjust it.